src/main/java/bennett/earthquake/EarthquakeFrame.java

```diff
@@ -0,0 +1,81 @@
+package bennett.earthquake;
+
+import bennett.earthquake.json.Feature;
+import bennett.earthquake.json.FeatureCollection;
+import hu.akarnokd.rxjava3.swing.SwingSchedulers;
+import io.reactivex.rxjava3.core.Flowable;
+import io.reactivex.rxjava3.disposables.Disposable;
+import io.reactivex.rxjava3.schedulers.Schedulers;
+
+import javax.swing.*;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class EarthquakeFrame extends JFrame {
+
+    private JList<String> jlist = new JList<>();
+    private ButtonGroup buttonGroup = new ButtonGroup();
+    private JRadioButton oneHourButton = new JRadioButton("One Hour");
+    private JRadioButton thirtyDaysButton = new JRadioButton("Thirty Days");
+
+    public EarthquakeFrame() {
+
+        setTitle("EarthquakeFrame");
+        setSize(300, 600);
+        setDefaultCloseOperation(EXIT_ON_CLOSE);
+
+        setLayout(new BorderLayout());
+
+        JPanel topPanel = new JPanel(new FlowLayout());
+        topPanel.add(oneHourButton);
+        topPanel.add(thirtyDaysButton);
+
+        buttonGroup.add(oneHourButton);
+        buttonGroup.add(thirtyDaysButton);
+
+        add(topPanel, BorderLayout.NORTH);
+        add(jlist, BorderLayout.CENTER);
+
+        oneHourButton.setSelected(true);
+
+        ActionListener radioButtonListener = new ActionListener() {
+            @Override
+            public void actionPerformed(ActionEvent e) {
+                if (oneHourButton.isSelected()) {
+                    fetchData(new EarthquakeServiceFactory().getService().oneHour());
```

Comment:
Create EarthquakeService only once

Reply:
how do i do that? 
also, i got flowable from chatgpt and then googled it and read up. i'm honestly still not so sure i get it, should i change it?

Reply:
as for part 4, i just committed something but its still not working and im struggling to figure out what im doing wrong/ what im missing

Reply:
> how do i do that? 
> also, i got flowable from chatgpt and then googled it and read up. i'm honestly still not so sure i get it, should i change it? 

I did not tell you to use a Flowable

Reply:
> as for part 4, i just committed something but its still not working and im struggling to figure out what im doing wrong/ what im missing 

Explain what goes wrong

Reply:
so should i get rid of flowable? 
what goes wrong is that its not connected to the google maps, so nothing happens when i press on it. i think i am missing stuff that should be in the feature class but im not sure what.

Reply:
what can i do instead of flowable because i coudlnt figure it out before

Reply:
> what can i do instead of flowable because i coudlnt figure it out before 

I literally gave you the code for it in my PR